buildspec/linuxTests.yml

```diff
@@ -13,11 +13,15 @@ env:
 phases:
   install:
     commands:
+      - export JAVA_HOME=/usr/lib/jvm/java-21-amazon-corretto.x86_64
```

Comment:
Should these 2 statements be under an if-else block?

Reply:
no

Reply:
this change has propagated to prod infra